<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1000</width>
				<height>600</height>
			</rect>
		</property>
		<property name="windowFlags">
			<set>Qt::FramelessWindowHint</set>
		</property>
		<property name="windowTitle">
			<string>全国计算机等级考试题库</string>
		</property>
		<property name="styleSheet">
			<string notr="true">
				QMainWindow {
				border: none;
				}
			</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<layout class="QVBoxLayout" name="verticalLayout_main">
				<property name="spacing">
					<number>0</number>
				</property>
				<property name="margin">
					<number>0</number>
				</property>
				<item>
					<widget class="QWidget" name="customTitleBar">
						<property name="styleSheet">
							<string notr="true">
								QWidget {
								background-color: #3498db;
								}
								QLabel {
								color: #ffffff;
								padding: 0px;
								font-size: 16px;
								}
								QPushButton {
								background-color: transparent;
								border: 5px solid transparent;
								min-width: 30px;
								min-height: 30px;
								}
								QPushButton:hover {
								background-color: #5dade2;
								}
								QPushButton#closeButton:hover {
								background-color: #ff0000;
								}
							</string>
						</property>
						<layout class="QHBoxLayout" name="horizontalLayout_titleBar">
							<property name="spacing">
								<number>0</number>
							</property>
							<property name="margin">
								<number>0</number>
							</property>
							<item>
								<widget class="QWidget" name="leftPlaceholder">
									<property name="minimumSize">
										<size>
											<width>80</width>
											<height>0</height>
										</size>
									</property>
									<property name="maximumSize">
										<size>
											<width>200</width>
											<height>1</height>
										</size>
									</property>
								</widget>
							</item>
							<item>
								<spacer name="leftSpacer">
									<property name="sizeType">
										<enum>QSizePolicy::Expanding</enum>
									</property>
								</spacer>
							</item>
							<item>
								<widget class="QLabel" name="titleLabel">
									<property name="text">
										<string>全国计算机等级考试题库</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<spacer name="middleSpacer">
									<property name="sizeType">
										<enum>QSizePolicy::Expanding</enum>
									</property>
								</spacer>
							</item>
							<item>
								<widget class="QPushButton" name="minimizeButton">
									<property name="text">
										<string></string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="maximizeButton">
									<property name="text">
										<string></string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="closeButton">
									<property name="text">
										<string></string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<layout class="QHBoxLayout" name="horizontalLayout">
						<property name="spacing">
							<number>0</number>
						</property>
						<property name="margin">
							<number>0</number>
						</property>
						<item>
							<widget class="QWidget" name="sidebar">
								<property name="minimumSize">
									<size>
										<width>80</width>
										<height>0</height>
									</size>
								</property>
								<property name="maximumSize">
									<size>
										<width>200</width>
										<height>16777215</height>
									</size>
								</property>
								<property name="styleSheet">
									<string notr="true">
										QWidget {
										background-color: #3498db;
										}
										QPushButton {
										background-color: transparent;
										border: none;
										padding: 10px;
										color: white;
										text-align: center;
										icon-size: 36px;
										width: 60px;
										}
										QPushButton:hover {
										background-color: #5dade2;
										}
										QPushButton:pressed {
										background-color: #2980b9;
										}
										QPushButton:checked {
										background-color: #2c3e50;
										}
									</string>
								</property>
								<layout class="QHBoxLayout" name="horizontalLayout_sidebar">
									<property name="spacing">
										<number>0</number>
									</property>
									<property name="margin">
										<number>0</number>
									</property>
									<item>
										<layout class="QVBoxLayout" name="verticalLayout">
											<property name="spacing">
												<number>0</number>
											</property>
											<property name="margin">
												<number>0</number>
											</property>
											<item>
												<widget class="QPushButton" name="btn_special">
													<property name="toolTip">
														<string>专项练习</string>
													</property>
													<property name="icon">
														<iconset>
															<normaloff>:/ExamBankSystem/resources/专项练习.png</normaloff>
														</iconset>
													</property>
													<property name="text">
														<string></string>
													</property>
													<property name="flat">
														<bool>true</bool>
													</property>
												</widget>
											</item>
											<item>
												<widget class="QPushButton" name="btn_exam">
													<property name="toolTip">
														<string>模拟考试</string>
													</property>
													<property name="icon">
														<iconset>
															<normaloff>:/ExamBankSystem/resources/考试.png</normaloff>
														</iconset>
													</property>
													<property name="text">
														<string></string>
													</property>
													<property name="flat">
														<bool>true</bool>
													</property>
												</widget>
											</item>
											<item>
												<widget class="QPushButton" name="btn_manage">
													<property name="toolTip">
														<string>考试环境管理</string>
													</property>
													<property name="icon">
														<iconset>
															<normaloff>:/ExamBankSystem/resources/虚拟机.png</normaloff>
														</iconset>
													</property>
													<property name="text">
														<string></string>
													</property>
													<property name="flat">
														<bool>true</bool>
													</property>
												</widget>
											</item>
											<item>
												<spacer name="verticalSpacer">
													<property name="orientation">
														<enum>Qt::Vertical</enum>
													</property>
													<property name="sizeHint" stdset="0">
														<size>
															<width>20</width>
															<height>40</height>
														</size>
													</property>
												</spacer>
											</item>
										</layout>
									</item>
									<item>
										<widget class="QWidget" name="sidebarHiddenPart">
											<property name="minimumSize">
												<size>
													<width>0</width>
													<height>0</height>
												</size>
											</property>
											<property name="maximumSize">
												<size>
													<width>120</width>
													<height>16777215</height>
												</size>
											</property>
											<property name="styleSheet">
												<string notr="true">
													QWidget {
													background-color: #3498db;
													}
													QPushButton {
													color: white;
													font-size: 16px;
													padding: 18px 0px;
													margin-left: -20px;
													}
												</string>
											</property>
											<layout class="QVBoxLayout" name="verticalLayout_hidden">
												<property name="spacing">
													<number>0</number>
												</property>
												<property name="margin">
													<number>0</number>
												</property>
												<item>
													<widget class="QPushButton" name="label_special">
														<property name="minimumSize">
															<size>
																<width>120</width>
																<height>0</height>
															</size>
														</property>
														<property name="text">
															<string>专项练习</string>
														</property>
													</widget>
												</item>
												<item>
													<widget class="QPushButton" name="label_exam">
														<property name="minimumSize">
															<size>
																<width>120</width>
																<height>0</height>
															</size>
														</property>
														<property name="text">
															<string>模拟考试</string>
														</property>
													</widget>
												</item>
												<item>
													<widget class="QPushButton" name="label_manage">
														<property name="minimumSize">
															<size>
																<width>120</width>
																<height>0</height>
															</size>
														</property>
														<property name="text">
															<string>考试环境管理</string>
														</property>
													</widget>
												</item>
												<item>
													<spacer name="verticalSpacer_hidden">
														<property name="orientation">
															<enum>Qt::Vertical</enum>
														</property>
														<property name="sizeHint" stdset="0">
															<size>
																<width>20</width>
																<height>40</height>
															</size>
														</property>
													</spacer>
												</item>
											</layout>
										</widget>
									</item>
								</layout>
							</widget>
						</item>
						<item>
							<widget class="QStackedWidget" name="mainContent">
								<property name="styleSheet">
									<string notr="true">
										QWidget {
										background-color: white;
										}
									</string>
								</property>
								<widget class="QWidget" name="defaultPage">
									<layout class="QVBoxLayout" name="verticalLayout_2">
										<property name="spacing">
											<number>0</number>
										</property>
										<property name="margin">
											<number>0</number>
										</property>
									</layout>
								</widget>
							</widget>
						</item>
					</layout>
				</item>
			</layout>
		</widget>
	</widget>
	<resources>
		<include location="ExamBankSystem.qrc"/>
	</resources>
	<connections/>
</ui>